tests/end-to-end/tests/accordion.cy.ts

```diff
@@ -30,6 +30,13 @@ describe('Accordion component', function () {
     cy.visit('/content/howlite-test/pages/Accordion.html');
 
     cy.percySnapshotWithAuth('Accordion preview');
+
+    cy.contains('Sample title').click();
+    cy.contains('Sample text')
+      .should('be.visible')
+      .parent()
```

Comment:
I think checking for visibility is enough.
For example - even if the html structure changes OR class name 'opened' will change to something else - all we want to know is that accordion content is visible - functionality works as expected